release-notes.txt

```diff
@@ -6,7 +6,9 @@ Date: 2021-10-06
 Add EdgeCore AS7726-32X (32 port 100G switch based on Trident3) as supported platform .
 
 ! Bug fixes
-For systemd-networkd, the redundant `/etc/systemd/network/99-enp.network` management interface configuration file was dropped. 
+For systemd-networkd, the redundant `/etc/systemd/network/99-enp.network` management interface configuration file was dropped.  Allows
+the pre-packaged `/lib/systemd/network/80-wired.network` file taking precendence. For upgrades from earlier version, the file in `/etc/systemd/network`
```

Comment:
"Allows the pre-packaged..."
This sentence does not make any sense, what do we mean?

Reply:
I mean in the sense that the sentence is not complete, do we mean "This allows..."?